lib/asciidoctor/converter/semantic_html5.rb

```diff
@@ -80,6 +80,32 @@ def convert_inline_anchor node
     end
   end
 
+  def convert_inline_quoted node
+    attributes = common_html_attributes node.id, node.role
+    case node.type
+    when :strong
+      %(<strong#{attributes}>#{node.text}</strong>)
+    when :emphasis
+      %(<em#{attributes}>#{node.text}</em>)
+    when :monospaced
+      %(<code#{attributes}>#{node.text}</code>)
+    when :mark
+      %(<mark#{attributes}>#{node.text}</mark>)
+    when :superscript
+      %(<sup#{attributes}>#{node.text}</sup>)
+    when :subscript
+      %(<sub#{attributes}>#{node.text}</sub>)
+    when :single
+      attributes = common_html_attributes node.id, node.role, 'single_quote'
```

Comment:
Not sure about this default role. If we want to be consistent, then it should be `singlequote` and `doublequote`: https://github.com/asciidoctor/asciidoctor/blob/feature/html-converter-next/devdocs/modern-html-converter/built-in-roles.adoc

Reply:
fixed with 21852ae